discord/client.py

```diff
@@ -1115,22 +1111,38 @@ def event(self, coro: Coro) -> Coro:
 
         .. code-block:: python3
 
-            @client.event
+            @client.event()
             async def on_ready():
                 print('Ready!')
 
+            @client.event(EventType.message)
+            async def hello_message_arrived(message):
+                if message.author != client.user:
+                    await message.reply("Nice")
+
+        Parameters
+        ------------
+        event_name: :type:`Union[str, EventType]`
+            The name of the event to register this coroutine under. If this is not provided, then
```

Comment:
Not necessarily the name of the event anymore
```suggestion
            The event to register this coroutine under. If this is not provided, then
```
This might fit better